src/passes/MinifyImportsAndExports.cpp

```diff
@@ -148,10 +148,6 @@ struct MinifyImportsAndExports : public Pass {
     std::map<Name, Name> oldToNew;
     std::map<Name, Name> newToOld;
     auto process = [&](Name& name) {
-      // do not minifiy special imports, they must always exist
-      if (name == MEMORY_BASE || name == TABLE_BASE || name == STACK_POINTER) {
```

Comment:
seems it minify `STACK_POINTER` as well. Is it supposed to be?

Reply:
Yes, the special stack pointer import is relevant only for dynamic linking, but as mentioned earlier, we don't run this pass in that case anyhow.

Reply:
Yes, I see